ocrd_tesserocr/ocrd-tool.json

```diff
@@ -24,6 +24,11 @@
           "default": "line",
           "description": "PAGE XML hierarchy level to add the TextEquiv results to (requires existing layout annotation up to one level above that)"
         },
+        "overwrite_lines": {
+          "type": "boolean",
+          "default": false,
+          "description": "remove existing layout and text annotation below the TextLine level"
```

Comment:
If it is "below" the text line level, pls. choose a different name, `overwrite_lines` suggests a new segmentation of regions into lines.

Reply:
Oh right, that did not occur to me. (I was more concerned about confusion with `textequiv_level`. This parameter is tied to the line level because the OCR itself is internally.) How about `overwrite_words`, despite the fact that glyphs will be removed as well? Or would `overwrite_line` be unambiguous enough?

Reply:
Both okay for me with a slight preference for `overwrite_words`.